test/helpers/dao.js

```diff
@@ -0,0 +1,62 @@
+const { hash } = require('eth-ens-namehash')
+const { getEventArgument } = require('@aragon/contract-test-helpers/events')
+
+const Kernel = artifacts.require('@aragon/core/contracts/kernel/Kernel')
```

Comment:
`@aragon/core` 👀

Never thought I'd see that again. We should switch to `@aragon/os` :).